extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -0,0 +1,848 @@
+# EXT\_feature\_metadata
+
+## Contributors
+
+* Sean Lilley, Cesium
+* Peter Gagliardi, Cesium
+* Bao Tran, Cesium
+* Sam Suhag, Cesium
+* Samuel Vargas, Cesium
+* Patrick Cozzi, Cesium
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec. Depends on [`EXT_mesh_gpu_instancing`](https://github.com/KhronosGroup/glTF/pull/1691) for instanced features.
+
+## Contents
+
+* Overview
+* Concepts
+    * Feature Classes
+    * Texture Metadata
+        * Per-texel Metadata
+        * Texel feature Ids
+    * Vertex Metadata
+        * Per-vertex Metadata
+        * Vertex feature IDs
+* Examples
+* JSON Schema
+
+
+## Overview
+
+**Features** are geometric entities in 3D space and may have application specific properties associated with them. These properties may span a myriad of domains, such as the heights of buildings in a city, the names of different structures in a CAD model, the classification of different points in a point cloud, or the age of different trees in a forest of instanced tree models. This extension adds a mechanism for storing an unlimited amount of feature metadata in a glTF asset and identifying features in the asset on a per-vertex or per-texel basis.
+
+## Concepts
+
+### Feature Classes
+
+### Feature Tables
+
+### Texture Metadata
+
+#### Per-texel Metadata
+
+#### Texel feature IDs
+
+### Vertex Metadata
+
+#### Per-vertex Metadata
+
+#### Vertex feature IDs
```

Comment:
TODO: Flesh these out. See if any material can be reused from the old spec (with minor rewordings) or if it's easier to start from scratch.